debian/memcached.default

```diff
@@ -1,2 +1,3 @@
 # Set this to no to disable memcached.
 ENABLE_MEMCACHED=yes
+START_PREFIX=/usr/share/clearwater/bin/run-in-signaling-namespace
```

Comment:
As discussed, this is pretty much a Clearwater-specific build of memcached (since the 1.6 branch is formally abandoned) so that's fine.